locales/es.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Back": "Espalda",
+  "Newer Posts": "Las publicaciones más nuevas",
+  "of": "de",
+  "Older Posts": "Publicaciones antiguas",
+  "Page": "Página",
+  "with the email address": "con la dirección de correo electrónico",
+  "Your email address": "Tu correo electrónico",
+  "You've successfully subscribed to": "Te has suscrito con éxito a",
+  "A collection of posts": "Una colección de publicaciones",
+  "A collection of 1 post": "Una colección de 1 publicación",
+  "A collection of % posts": "Una colección de % publicaciones",
+  "Get the latest posts delivered right to your inbox": "Recibe las últimas publicaciones entregadas directamente en tu bandeja de entrada",
+  "Latest Posts": "Últimas publicaciones",
+  "No posts": "No publicaciones",
+  "Read More": "Lee mas",
+  "See all % posts": "Ver las & publicaciones",
+  "Share this": "Compartir este",
+  "Stay up to date! Get all the latest & greatest posts delivered straight to your inbox": "¡Estar al día! Obtenga todas las publicaciones más recientes y mejores entregadas directamente en su bandeja de entrada",
+  "Subscribe": "Suscribir",
+  "Subscribe to": "Suscribirse a",
+  "Subscribed!": "¡Suscrito!",
+  "Subscribe via RSS!": "Suscríbete con RSS!",
+  "Subscribe to": "Suscríbete a ",
+  "All rights reserved.": "Todos los derechos reservados.",
+  "youremail@example.com": "tucorreoelectronico@example.com",
```

Comment:
Probably `su.email@example.es`, though `.com` is fine too.

Reply:
We're using the proper name elsewhere so we could do `sucorreoelectronico@...` or the less formal `su.email@...`